app.py

```
@@ -408,23 +546,72 @@ def webhook():
+                        if message_type == 'text':
+                            text_body = msg.get('text', {}).get('body', '')
+                        elif message_type in {'image', 'video', 'audio', 'document'}:
+                            media_info = msg.get(message_type, {})
+                            media_id = media_info.get('id')
+                            text_body = media_info.get('caption', '')
+                            file_name = media_info.get('filename') if message_type == 'document' else None
+                            media_mime_type = media_info.get('mime_type')
+
+                            if media_id and whatsapp_config_ready():
+                                try:
+                                    downloaded_path, stored_name, mime_type = download_whatsapp_media(media_id, file_name)
```

Ruff (0.14.14)

[warning] Unpacked variable `downloaded_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)